### PR TITLE
Corrected name of staging and prod deploy log

### LINKED ISSUE #2708: Feedback on page: /guides/v2.2/cloud/trouble/environments-logs.html
https://devdocs.magento.com/guides/v2.2/cloud/trouble/environments-logs.html
please look into the section 

Logs from the deploy hook are located on the server in the following locations:

Integration: /var/log/deploy.log
Staging: /var/log/platform/<project ID>_stg/post_deploy.log
Production: /var/log/platform/<project ID>/post_deploy.log

the post_deploy.log is not longer used. Now the deploy is  logged in the file: 

Staging: /var/log/platform/<project ID>_stg/deploy.log
Production: /var/log/platform/<project ID>/deploy.log

### PR DESCRIPTION
**Pull request by mage2pratik**
Fixes #2708 
## This PR is a:

- [ ] New topic
- [ ] Content update
- [x] Content fix or rewrite
- [ ] Bug fix or improvement

## Summary

Corrected the name of the deployment log for Magento Commerce Cloud staging and production environments.

## Additional information

**Affected URLs**

- https://devdocs.magento.com/guides/v2.1/cloud/project/project-start.html
- https://devdocs.magento.com/guides/v2.2/cloud/project/project-start.html
- https://devdocs.magento.com/guides/v2.1/cloud/project/project-start.html

whatsnew
Corrected the name of the {{site.data.var.ece}} Staging and Production deploy log.

### CODE REVIEW

**Comment by magento-cicd2** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by meker12** [comment]
This is a regression. Content was fixed earlier in https://github.com/magento/devdocs/pull/2802.

**Comment by meker12** [comment]
running tests

**Comment by meker12** [comment]
@mage2pratik  Thanks for your updates!

File: guides/v2.2/cloud/project/project-start.md [hunk 1]
```diff
@@ -73,16 +73,16 @@ After pushing changes to your environment, you can observe the build results in
 You can review the logs from the deploy hook in the following directories:
 
 -  **Integration**: `/var/log/deploy.log`
--  **Staging**: `/var/log/platform/<project-ID>_stg/post_deploy.log`
--  **Production**: `/var/log/platform/<node-{1|2|3}>.<project-ID>/post_deploy.log`
+-  **Staging**: `/var/log/platform/<project-ID>_stg/deploy.log`
```
Comment (hguthrie) [inline guides/v2.2/cloud/project/project-start.md]: @meker12  Please take a look at this wiki and confirm these changes: https://wiki.corp.magento.com/display/~yyevtushenko/Magento+Cloud+logs

**Comment by dshevtsov** [comment]
@meker12 , Liquid variables don't work in whatsnew.

**Pull request merged**